### PR TITLE
encoder counts

### PR DESCRIPTION
**Pull request by nutansg**

### CODE REVIEW

**Comment by EpicWolverine** [comment]
You may want to name the counts as “channel 1” and “channel 2” instead of “left” and “right” to make the code more reusable.

**Comment by mattjay1024** [comment]
> You may want to name the counts as “channel 1” and “channel 2” instead of “left” and “right” to make the code more reusable.

I don't think we have motor controllers with more than 2 channels, or if roboteq even makes those

File: isc_shared_msgs/msg/EncoderCounts.msg [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Header header
-int32 left_count
-int32 right_count
+int left_count
```
Comment (mattjay1024) [inline isc_shared_msgs/msg/EncoderCounts.msg]: left_count and right_count should both be of the int32 type

**Comment by EpicWolverine** [comment]
I meant that a different robot may have the channels and left/right swapped, but I suppose the left/eight naming convention is consistent with the existing wheel speeds naming. I think it’s fine if it’s consistent with that then.

**Pull request merged**